### PR TITLE
Add stateful class MultiSelectDropdown shared component, with Select Roles story

### PR DESCRIPTION
**Pull request by cryptoquick**
Also includes some Storybook fixes I had stashed.

Test with `npm run storybook` and visit <http://localhost:6006/?selectedKind=MultiSelectDropdown&selectedStory=Select%20Roles>

### CODE REVIEW

File: ui/src/shared/components/MultiSelectDropdown.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    const {localSelectedItems, isOpen} = this.state
+    const {label} = this.props
+    const labelText = isOpen ? "0 Selected" : "Apply Function"
+
+    return (
+      <div className={classNames('dropdown multi-select-dropdown', {open: isOpen})}>
+        <div onClick={::this.toggleMenu} className="btn btn-xs btn-info dropdown-toggle" type="button">
+          <span className="multi-select-dropdown__label">
+            {
+              label ?
```
Comment (jaredscheib) [inline ui/src/shared/components/MultiSelectDropdown.js]: Your linter should be yelling at you about nested ternary expressions. I'm cool with them but wonder if this indicates a useful refactor?

Comment (cryptoquick) [inline ui/src/shared/components/MultiSelectDropdown.js]: good call

File: ui/stories/admin.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        label={'Select Roles'}
+        onApply={action('onApply')}
+      />
+    </Center>
+  ))
+
+storiesOf('Tooltip', module)
+  .add('Delete', () => (
+    <Center>
+      <Tooltip tip={`Are you sure? TrashIcon`}>
```
Comment (jaredscheib) [inline ui/stories/admin.js]: `TrashIcon` rendered as text here -- should it be?

Comment (cryptoquick) [inline ui/stories/admin.js]: that's just for testing, it's in a story instance
in another branch I replaced it with an emoji tho
I'd have to do some digging to find the actual icon...

Comment (jaredscheib) [inline ui/stories/admin.js]: ah ok no worries

**Comment by jaredscheib** [comment]
Looks great! Nice work, @cryptoquick!

**Pull request merged**